src/channel.rs

```diff
@@ -51,13 +43,7 @@ pub struct Package {
 }
 
 pub fn is_beta_toolchain(name: &str) -> bool {
```

Comment:
Should we rename this into something like `is_reserved_channel`? Given that we do not have any beta networks right now.

```suggestion
pub fn is_reserved_toolchain(name: &str) -> bool {
```

Reply:
I changed it to `is_dateless_distributed_toolchain` since "reserved" doesn't fully describe it